### DISCUSSION TITLE
How does one specify the `library` field using Pandoc export

### DISCUSSION THREAD

**Discussion by ralmond** [retorquere/zotero-better-bibtex#2704] [Q&A] [answered]
After reading the documentation, I'm not sure how to specify the `library:` I want to export.  I currently have a shared group `PhysicsPlayground` and within it a collection `ProjectPapers`.  It is unclear how to specify the YAML header for the input file.

I've tried:
```
---
zotero:
  library: PhysicsPlayground/ProjectPapers
  csl-style: apa
---
```

I've tried replacing that with the name of one of my local (not shared projects).

In both cases, I'm getting the following:
```
bash-3.2$ pandoc -s --lua-filter=zotero.lua papers-in.md -o papers-out.md
zotero-live-citations f47be20
```
The `papers-out.md` has just the YAML header, but no citations.

I think that the instructions could use an example or two of how to specify the value for the `library` field.

**Comment by retorquere** [comment, marked as answer]
You can only specify the library, not the collection within it.
